src/kernel/mconfig.ml

```diff
@@ -133,7 +134,12 @@ let dump_merlin x =
     );
     "stdlib"       , Json.option Json.string x.stdlib;
     "unit_name"    , Json.option Json.string x.unit_name;
-    "wrapping_prefix" , Json.option Json.string x.wrapping_prefix;
+    "unit_name_for", (let alist =
+                        x.unit_name_for
+                        |> String.Map.map ~f:Json.string
```

Comment:
I don't know what `Json.string` is doing, but it seems this is running it only on the values, not the keys.  Is that what we want?

Reply:
`Json.string` = `fun x -> `String x`. This is what we want - an `'Assoc` expects a `(string * Json.t) list`, not a `(Json.t * Json.t) list`.